sdcm/cluster_k8s/eks.py

```diff
@@ -423,19 +429,16 @@ def create_eks_cluster(self, wait_till_functional=True):
             },
             tags=self.tags,
         )
+
```

Comment:
Need to add some info about this change to the commit description.

Why this update is needed?